editor/editor_audio_buses.cpp

```diff
@@ -1077,14 +1077,16 @@ void EditorAudioBuses::_load_layout() {
 
 void EditorAudioBuses::_load_default_layout() {
 
-	Ref<AudioBusLayout> state = ResourceLoader::load("res://default_bus_layout.tres");
+	String layout_path = ProjectSettings::get_singleton()->get("audio/default_bus_layout");
+
+	Ref<AudioBusLayout> state = ResourceLoader::load(layout_path);
 	if (state.is_null()) {
-		EditorNode::get_singleton()->show_warning(TTR("There is no 'res://default_bus_layout.tres' file."));
+		EditorNode::get_singleton()->show_warning(TTR("There is no '" + layout_path + "' file."));
```

Comment:
```suggestion
		EditorNode::get_singleton()->show_warning(vformat(TTR("There is no '%s' file."), layout_path));
```
Otherwise it is not translatable.

Reply:
Makes sense, din't think of that.